### PR TITLE
Revision of snapshotTime handling in HLT and new Run2 MC GT with ES fixes

### PR DESCRIPTION
**Pull request by diguida**
# Summary of Changes for HLT Testing

Revision of snapshotTime handling in HLT: no longer use/set a global `snapshotTime` but move to a per-payload `snapshotTime`, set to the time of PR #12095 
# Summary of Changes in the GT

Originally in #12114 commit https://github.com/mmusich/cmssw/commit/bee66c2e63f42f434e9cd95740d37c49a7e6ffef
## RunII Simulation
- RunII Ideal scenario : 76X_mcRun2_design_v11 as 76X_mcRun2_design_v10 with the following changes:
  - updated Preshower pedestals as per pedestal run in low gain taken in Run2015D
  - updated Preshower Zero suppression thresholds with 6 ADC
- RunII Asymptotic scenario : 76X_mcRun2_asymptotic_v11 as 76X_mcRun2_asymptotic_v10 with the following changes:
  - updated Preshower bad channels as per Run2015D
  - updated Preshower pedestals as per pedestal run in low gain taken in Run2015D
  - updated Preshower Zero suppression thresholds with 6 ADC
  - RunII Startup scenario : 76X_mcRun2_startup_v11 as 76X_mcRun2_startup_v10 with the following changes:
  - updated Preshower bad channels as per Run2015D
  - updated Preshower pedestals as per pedestal run in low gain taken in Run2015D
  - updated Preshower Zero suppression thresholds with 6 ADC
- RunII Heavy Ions scenario : 76X_mcRun2_HeavyIon_v11 as 76X_mcRun2_HeavyIon_v10 with the following changes:
  - updated Preshower bad channels as per Run2015D
  - updated Preshower pedestals as per pedestal run in low gain taken in Run2015D
  - updated Preshower Zero suppression thresholds with 6 ADC

### CODE REVIEW

**Comment by diguida** [comment]
please test

**Comment by diguida** [comment]
@cmsbuild please acknowledge changes

**Comment by davidlange6** [comment]
@cmsbuild, please test
I will close #12114

**Comment by diguida** [comment]
please test

File: Configuration/HLT/python/autoCondHLT.py [hunk 1]
```diff
@@ -1,24 +1,30 @@
 # https://cms-conddb.cern.ch/browser/#search
 
+#default value for all L1T menus
+connectionString = "frontier://FrontierProd/CMS_CONDITIONS"
+l1MenuRecord = "L1GtTriggerMenuRcd"
+l1MenuLabel = ""
+
+#The snapshot time has been set as starting point as the one of PR 12096.
```
Comment (Martin-Grunewald) [inline Configuration/HLT/python/autoCondHLT.py]: This should be 12095 (not 12096)...

Comment (diguida) [inline Configuration/HLT/python/autoCondHLT.py]: oh, typo...
Thanks for spotting this.

**Comment by diguida** [comment]
Interesting: the bot sees the failure (timeout) [here](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-12167/9331/runTheMatrix-results/matrixTests.log), but the latest test is actually [fine](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-12168/9332/runTheMatrix-results/runall-report-step123-.log).

**Comment by diguida** [comment]
@emanueledimarco @shervin86
please give a look  [here](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_7_6_X_2015-10-28-2300+12168/11011/validateJR/all_OldVSNew_TTbarPUwf50202p0/) ([sample plot](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_7_6_X_2015-10-28-2300+12168/11011/validateJR/all_OldVSNew_TTbarPUwf50202p0/all_OldVSNew_TTbarPUwf50202p0c_log10EcalRecHitsSorted_ecalPreshowerRecHit_EcalRecHitsES_RECO_obj_obj_energy.png)) and [here](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_7_6_X_2015-10-28-2300+12168/11011/validateJR/all_OldVSNew_TTbarPUwf25202p0) ([sample plot](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_7_6_X_2015-10-28-2300+12168/11011/validateJR/all_OldVSNew_TTbarPUwf25202p0/all_OldVSNew_TTbarPUwf25202p0c_log10EcalRecHitsSorted_ecalPreshowerRecHit_EcalRecHitsES_RECO_obj_obj_energy.png)) and confirm that the changes of the ES conditions:

* bad channels
* pedestals
* ZS thresholds

go in the right direction.

**Comment by shervin86** [comment]
Adding ES experts: Arabella and Ming.

-- 
 							Shervin

**Comment by diguida** [comment]
@cmsbuild please acknowledge changes

**Comment by diguida** [comment]
please test

**Comment by shervin86** [comment]
The changes are expected, more low energy recHits as due to the lower ZS threshold and realistic noise per each strip

Arabella

**Comment by diguida** [comment]
@shervin86 thanks for confirming.
As soon as tests are ok, they can go in.

**Comment by davidlange6** [comment]
merging and will look at the comparisons once they finish..

**Comment by Martin-Grunewald** [comment]
+1

**Pull request merged**